app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,49 @@
+class FriendshipsController < ApplicationController
+  def new
+    @users = User.all_except(current_user)
+  end
+
+  def create
+    user = current_user
+      @friend = Friendship.create(user_id: user.id, friend_id: params[:friend_id])
+      if @friend.save
+        flash[:success] = "Request sent successfully"
+        redirect_to new_friendship_path
+      else
+        flash[:error] = "Already Friends"
+        redirect_to new_friendship_path
+      end
+  end
+
+  def show
+    @friend = Friendship.find(params[:id])
+  end
+
+  def edit
+    user = current_user
+    @friend = Friendship.find_by_user_id_and_friend_id(params[:id], user.id)
+  end
+  
+  def update
```

Comment:
`update` is a general method, it's not appropirate to use it for accepting friend request. Consider in future we support option for unfriend, how we are going to do it?

Best approach here is to make member routes 
- `friendship/:id/accept`
- `friendship/:id/reject`
- `friendship/:id/unfriend`
etc

Reply:
reject/ remove friend using destroy method